web/tests/Web.E2ETests/Steps/School/HomeSteps.cs

```diff
@@ -197,4 +197,11 @@ public async Task ThenTheRagGuidanceIsVisible()
         Assert.NotNull(_schoolHomePage);
         await _schoolHomePage.AssertRagGuidance();
     }
+
+    [Then("I should see a 404 error page indicating the BRC is not available for this school")]
+    public async Task xxx()
```

Comment:
![image](https://github.com/user-attachments/assets/56f12e03-6656-4021-b3e1-1c3f3c643376)

Reply:
🫣

Reply:
@jrabbott are you referring to the wording of the step?

Reply:
It's been resolved. The method name was triple x.